ports/mimxrt10xx/boards.c

```diff
@@ -101,6 +101,9 @@ void board_init(void)
 
   LPUART_Init(UART_DEV, &uart_config, freq);
 #endif
+
+  // Flash needs to be initialized to check for a valid image
+  board_flash_init();

```

Comment:
imxrt doesn't need this to call `board_flash_init()` to get it work

Reply:
It is safer to initialize the flexspi in board_init.  Flexspi is not initialized when using serial downloader mode, so doing this here will make the behavior consistent in serial downloader mode and internal boot mode.